src/css/states/states.less

```diff
@@ -0,0 +1,189 @@
+@import "../imports/vars";
+@import "../imports/icons";
+
+// idle
+
+.jw-state-idle {
+  .jw-icon-display {
+    .jw-icon-play;
```

Comment:
It could be useful to consolidate the location where we set the icon to use.  E.G. if we did this like `.jw-state-idle, .jw-state-paused { .jw-icon-display { .jw-icon-play; }}` it'd be easier to track down and manage these.